p2p/p2p.go

```diff
@@ -155,6 +155,10 @@ type Messenger interface {
 	// peer, but reuses a connection and a stream if possible.
 	SendToConnectedPeer(topic string, buff []byte, peerID PeerID) error
 
+	IsConnectedToTheNetwork() bool
```

Comment:
IsConnected would be enough

Reply:
We already have the function `IsConnected(peerID PeerID) bool`